src/OrbitGl/CaptureViewElement.h

```diff
@@ -22,9 +22,15 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
  public:
   explicit CaptureViewElement(CaptureViewElement* parent, TimeGraph* time_graph,
                               orbit_gl::Viewport* viewport, TimeGraphLayout* layout);
+
+  struct DrawContext {
```

Comment:
I like the `DrawContext` :-)

Reply:
Me too! But the name was a proposal from our TL.

Reply:
Team cooperation :-)